cli/cli_app.go

```diff
@@ -416,6 +418,15 @@ func RunTerragrunt(terragruntOptions *options.TerragruntOptions) error {
 		return runRenderJSON(terragruntOptions, terragruntConfig)
 	}
 
+	if shouldPrintModuleGroups(terragruntOptions) {
+		js, err := runGraphDependenciesGroups(terragruntOptions)
+		if err != nil {
+			return err
+		}
+		fmt.Printf("%s\n", js)
```

Comment:
To write to stdout, should be used TerragruntOptions.Writer 

Example: https://github.com/gruntwork-io/terragrunt/blob/master/cli/cli_app.go#L390

`fmt.Printf()` is not used in other places in code base

Reply:
Updated :)